base/compiler/ssair/passes.jl

```diff
@@ -888,7 +889,7 @@ function lift_keyvalue_get!(compact::IncrementalCompact, idx::Int, stmt::Expr, 
     compact[idx] = lifted_val === nothing ? nothing : Expr(:call, Core.tuple, lifted_val.val)
     if lifted_val !== nothing
         if !⊑(𝕃ₒ, compact[SSAValue(idx)][:type], result_t)
```

Comment:
```suggestion
        if !⊑(𝕃ₒ, compact[SSAValue(idx)][:type], tuple_tfunc(𝕃ₒ, Any[result_t]))
```
It looks like it's invalid to use `result_t` here? `compact[SSAValue(idx)][:type]` would be something like `Union{Nothing, Tuple{...}}`.